src/App.js

```diff
@@ -1,26 +1,33 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import data from './api/movies.json';
 
-export class App extends Component {
-  state = {
-    movies: data,
-  };
+export const App = () => {
+  const [movies, setMovies] = useState(data);
+
+  const addMovie = (newMovie) => {
+    const { imdbUrl } = newMovie;
+    const coincidence = movies.some(movie => (
+      movie.imdbUrl === imdbUrl
+    ));
 
-  render() {
-    const { movies } = this.state;
+    if (coincidence) {
+      return;
+    }
```

Comment:
![image](https://user-images.githubusercontent.com/57598825/122019007-0b355880-cdcc-11eb-9d01-13bad2095013.png)

Reply:
ahahahaha)))
You make my morning =)